nibabel/viewers.py

```diff
@@ -61,6 +61,9 @@ def __init__(self, data, affine=None, axes=None, title=None):
         title : str or None, optional
             The title to display. Can be None (default) to display no
             title.
+        vlim : array-like or None, optional
```

Comment:
Hey @matthew-brett. Sorry, busy times.

To clarify my usage when I was working on this: I would have multiple files with different ranges. By setting `vlim`, I would be able to directly compare them, both the images and the time series. So for my case, I definitely think keeping them the same was preferable.

However, if what we want is a separate `ylim`, we can do that. In which case, should we change `vlim` to `clim` to drive home that one is for images, one is for time series? I chose `vlim` initially specifically to abstract from `y/c`.

What do you think?